src/app/core/components/marios-list/marios-list.component.html

```diff
@@ -0,0 +1,13 @@
+<div class="container-fluid d-flex no-padding">
+  <div class="row">
+    <div class="col-md-12 mb-2">
+      <h4>{{ label | uppercase }}</h4>
+    </div>
+    <div class="col-md-4 mb-4" *ngFor="let mariosElement of mariosList.slice(0, 9)">
```

Comment:
Do you need to slice?

Reply:
Sure, because at this moment, I do not have pagination on this endpoint, and we do not want to have 65 rows of Marios. So, I want to have 9 relevant ones